deploy/ingress-serviceaccount.yaml

```diff
@@ -31,7 +31,13 @@ rules:
   - configmaps
   verbs:
   - get
-
+- apiGroups:
+  - zalando.org
+  resources:
+  - routegroups
+  verbs:
+  - get
+  - list
```

Comment:
It’s not enough!
We also need update/patch for subresource `status`, similar to ingress

Reply:
Thanks, I missed that :(